src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -9220,6 +9237,10 @@ function PSCreateDirectoryOnRemoteSession
                 else
                 {{
                     Microsoft.PowerShell.Management\New-Item $createDirectoryPath -ItemType Directory | Out-Null
+                    if (ExperimentalFeature.IsEnabled('PSCopyItemDirectoryAttributes'))
+                    {{
+                        (Get-Item $createDirectoryPath -Force).Attributes = $attributes
+                    }}
```

Comment:
Same here, when the remote side is an older version of PS, `ExperimentalFeature.IsEnabled('PSCopyItemDirectoryAttributes')` won't work.